tests/console/installation_snapshots.pm

```diff
@@ -0,0 +1,20 @@
+use base "consoletest";
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    become_root();
+
+    script_run("snapper list | tee /dev/$serialdev");
+    # Check if the snapshot called 'after installation' is there
+    $self->result('fail') unless wait_serial("after installation", 5);
```

Comment:
btw. $self->result setting is unnecessary here since wait_serial automatically updates it.